src/ContactForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './App.css';
+
+const ContactForm = (props) => {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState('')
+  const [profilePicture, setProfilePicture] = useState('')
+
+  const handleClick = () => {
+    props.addContact({
+      name,
+      email,
+      phoneNumber,
+      profilePicture
+    })
+
+    props.history.push('/contacts')
+  }
+
+  ContactForm.propTypes = {
```

Comment:
`name`, `phoneNumber` and `email` are not props on `ContactForm` - the only prop i see `ContactForm` getting (or using) is `history`.